content/webapp/components/WorkDetails/index.tsx

```diff
@@ -120,7 +120,10 @@ const WorkDetails: FunctionComponent<Props> = ({
 
   const seriesPartOfs = work.partOf.filter(p => !p.id);
 
-  const physicalItems = getItemsWithPhysicalLocation(work.items ?? []);
+  const physicalItems = useMemo(
```

Comment:
This didn't have the effect I was expecting of making the child component re-render (as we maybe expected from [watching this](https://www.youtube.com/watch?v=epwue_cHsA8&ab_channel=AkileshRao)), but it did appear to make the component less janky, so I left it in